packages/destination-actions/src/destinations/the-trade-desk-crm/properties.ts

```diff
@@ -30,50 +30,36 @@ export const pii_type: InputField = {
 export const email: InputField = {
   label: 'User Email',
   description: "The user's email address to send to The Trade Desk.",
-  type: 'hidden', // This field is hidden from customers because the desired value always appears at path '$.context.traits.email' in Personas events.
+  type: 'string',
   default: {
     '@path': '$.context.traits.email'
-  }
+  },
+  readOnly: true
 }
 
 export const event_name: InputField = {
   label: 'Event Name',
   description: 'The name of the current Segment event.',
-  type: 'hidden', // This field is hidden from customers because the desired value always appears at path '$.event' in Personas events.
+  type: 'hidden',
```

Comment:
```suggestion
  type: 'string',
  unsafe_hidden: true,
```
I'll be migrating usages of `type: hidden` at some point so we can just make this change now